### PR TITLE
Fix 22 columns problem.

### PR DESCRIPTION
**Pull request by djx314**
Fix 22 columns problem and add some more than 22 columns case class with some fields have default value for test.

### CODE REVIEW

File: src/test/scala/slick/ext/repo.scala [hunk 1]
```diff
@@ -3,9 +3,34 @@ package slick.ext
...
+  
 }
+
+object TestCase extends App {
+  
+  object UserRepo extends UserRepo
+  
+  
+  import UserRepo._
+  import UserRepo.profile.simple._
```
Comment (jilen) [inline src/test/scala/slick/ext/repo.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/jilen-zhang/slickext/file/583337849/issues/source?bid=2273443&fileBranchId=2273469#l28)

File: src/test/scala/slick/ext/repo.scala [hunk 2]
```diff
@@ -3,9 +3,34 @@ package slick.ext
 import slick.ext.macros._
 
 trait UserRepo {
-  val profile: scala.slick.driver.JdbcProfile
+  //val profile: scala.slick.driver.JdbcProfile
+  val profile = scala.slick.driver.H2Driver
   import profile.simple._
+  import scala.slick.collection.heterogenous._
```
Comment (jilen) [inline src/test/scala/slick/ext/repo.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/jilen-zhang/slickext/file/583337849/issues/source?bid=2273443&fileBranchId=2273469#l9)

File: src/test/scala/slick/ext/repo.scala [hunk 3]
```diff
@@ -3,9 +3,34 @@ package slick.ext
...
+  class TestUserTable
+  
 }
+
+object TestCase extends App {
+  
+  object UserRepo extends UserRepo
+  
+  
+  import UserRepo._
```
Comment (jilen) [inline src/test/scala/slick/ext/repo.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://www.codacy.com/app/jilen-zhang/slickext/file/583337849/issues/source?bid=2273443&fileBranchId=2273469#l27)

**Comment by jilen** [comment]
已手工合并...

**Comment by djx314** [comment]
看你的那段代码感觉清晰多了...

**Pull request closed without merging**